project/plugins.sbt

```diff
@@ -14,3 +14,6 @@ addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.8.2")
 addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.0.0")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.2.0")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.2.2")
+
+libraryDependencies += "org.scala-sbt" % "scripted-plugin" % sbtVersion.value
```

Comment:
Yes this enables testing of the plugin, but the test I added is for this specific fix. You want me to separate that into another pr with a project that doesn't test this issue first, just a simple test that the plugin generates a scala file?